src/main/java/it/pagopa/pn/paperchannel/utils/PaperCalculatorUtils.java

```diff
@@ -51,12 +50,16 @@ public Mono<CostWithDriver> calculator(List<AttachmentInfo> attachments, Address
      * @param productType tipo di prodotto (AR, 890, etc)
      * @param isReversePrinter true, se il printType della request della SEND è di tipo BN_FRONTE_RETRO
      *
-     * @return calcolo del costp della notifica con le informazioni del recapitista
+     * @return calcolo del costo della notifica con le informazioni del recapitista
      */
     private Mono<CostWithDriver> getAmount(List<AttachmentInfo> attachments, String cap, String zone, String productType, boolean isReversePrinter){
         String processName = "Get Amount";
         log.logStartingProcess(processName);
 
+        //Simplified tender flow
+        if(pnPaperChannelConfig.isEnableSimplifiedTenderFlow()) {
+            return Mono.empty();
```

Comment:
@pasqualino-cristaudo-acn possiamo mettere un //TODO per ricordarci che deve essere implementato l'algoritmo e questa riga è solo temporanea?

Reply:
Sì certamente, una volta implementato il nuovo flusso il TODO verrà rimosso.
Sono stati aggiunti i log all'ingresso ed in uscita dell'if come da richiesta.